components/slack/sources/common/base.mjs

```diff
@@ -1,21 +1,77 @@
 import slack from "../../slack.app.mjs";
+import {
+  NAME_CACHE_MAX_SIZE, NAME_CACHE_TIMEOUT,
+} from "./constants.mjs";
 
 export default {
   props: {
     slack,
-    nameCache: "$.service.db",
+    db: "$.service.db",
   },
   methods: {
-    async maybeCached(key, refreshVal, timeoutMs = 3600000) {
-      let record = this.nameCache.get(key);
+    cleanCache(cacheObj) {
```

Comment:
We usually put in getter/setter methods for any db variables.
```
    _getNameCache() {
      return this.db.get("nameCache") ?? {};
    },
    _setNameCache(nameCache) {
      this.db.set("nameCache", nameCache);
    },
    _getLastCacheCleanup() {
      return this.db.get("lastCacheCleanup") ?? 0;
    },
    _setLastCacheCleanup(LastCacheCleanup) {
      this.db.set("LastCacheCleanup", LastCacheCleanup);
    },
```

Reply:
Thanks, added them!